dumper/app/build.gradle

```diff
@@ -92,6 +94,19 @@ dependencies {
     sources "ch.qos.logback:logback-classic:${libs.versions.logback.get()}@sources"
 }
 
+jacoco {
+    toolVersion = "0.8.11"
+}
+
+jacocoTestReport {
+    dependsOn test
+    reports {
+        xml.required = false
+        csv.required = false
+        html.outputLocation = layout.buildDirectory.dir('jacocoHtml')
```

Comment:
Conventionally, into the reports directory defined by the reporting plugin. But no matter.

Reply:
I can see the `dumper/app/build/reports` directory that contains a `tests` directory only. Maybe we can add jacoco to this `reports` directory (next to `tests`).

Reply:
Done, the output is now in the `reports` subdirectory.